src/storage/src/hummock/value.rs

```diff
@@ -184,7 +184,7 @@ mod tests {
     #[test]
     fn test_vec_decode_encode() {
         let mut result = vec![];
-        let value_meta = ValueMeta { vnode: 63492 };
+        let value_meta = ValueMeta { vnode: 233 };
```

Comment:
Do we still have vnode in value meta now?

Reply:
Will remove soon. Check #3316